.teamcity/src/main/kotlin/configurations/Gradleception.kt

```diff
@@ -20,23 +26,61 @@ class Gradleception(model: CIBuildModel, stage: Stage) : BaseGradleBuildType(sta
         javaCrash = false
     }
 
+    params {
+        // Override the default commit id so the build steps produce reproducible distribution
+        param("env.BUILD_COMMIT_ID", "HEAD")
```

Comment:
This ends up like this in the build scan as well, right? I suppose we need to set this since it ends up in the `build-receipt.properties`? Though that is only important for the first build when we use the distribution for hashing, right? So the following build could use the correct commit id again I suppose.